### PR TITLE
docs: Add README for getting started with rkt.

### PR DESCRIPTION
**Pull request by yifan-gu**
/cc @dchen1107 @vmarmol @jonboulle @bakins

### CODE REVIEW

**Comment by dchen1107** [comment]
Thanks for doing this. This is in my work queue for a while, but never get around to do it. 

LGTM.

File: docs/getting-started-guides/rkt/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Run Kubernetes with rkt
+
+This document describes how to run Kubernetes using [rkt](https://github.com/coreos/rkt) as a container runtime.
+We still have [a bunch of work](https://github.com/GoogleCloudPlatform/kubernetes/issues/8262) to do to make the experience with rkt wonderful, please stay tuned!
+
+### **Prerequisite**
+
+- [systemd](http://www.freedesktop.org/wiki/Software/systemd/) should installed on your machine and should be enabled. The minimum version required at this moment (2015/05/28) is [215](http://lists.freedesktop.org/archives/systemd-devel/2014-July/020903.html).
```
Comment (jonboulle) [inline docs/getting-started-guides/rkt/README.md]: s/should installed/should be installed/

File: docs/getting-started-guides/rkt/README.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  *(Note that systemd is not required by rkt itself, we are using it here to monitor and manage the pods launched by kubelet.)*
+
+- Install the latest rkt release according to the instructions [here](https://github.com/coreos/rkt).
+  The minimum version required for now is [v0.5.6](https://github.com/coreos/rkt/releases/tag/v0.5.6).
+
+- Make sure the `rkt metadata service` is running because it is necessary for running pod in private network mode.
+  More details about the networking of rkt can be found in the [document](https://github.com/coreos/rkt/blob/master/Documentation/networking.md).
```
Comment (jonboulle) [inline docs/getting-started-guides/rkt/README.md]: s/document/documentation/

File: docs/getting-started-guides/rkt/README.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+### Local cluster
+
+To use rkt as the container runtime, you just need to set the environment variable `CONTAINER_RUNTIME`:
+```shell
+$ export CONTAINER_RUNTIME=rkt
+$ hack/local-up-cluster.sh
+```
+
+After this, you can launch some pod in another terminal:
```
Comment (jonboulle) [inline docs/getting-started-guides/rkt/README.md]: s/pod/pods/

File: docs/getting-started-guides/rkt/README.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+```shell
+$ cluster/kubectl.sh create -f example/pod.yaml
+```
+
+### CoreOS cluster on GCE
+
+To use rkt as the container runtime for your CoreOS cluster on GCE, you need to specify the os distribution, project, image:
```
Comment (jonboulle) [inline docs/getting-started-guides/rkt/README.md]: s/os/OS/

File: docs/getting-started-guides/rkt/README.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+$ export KUBE_GCE_MINION_IMAGE=<image_id>
+$ export KUBE_GCE_MINION_PROJECT=coreos-cloud
+```
+
+Then you can launch the cluster by:
+````shell
+$ kube-up.sh
+```
+
+Note that we are still working on making all containerized the master component run smoothly in rkt. Before that we are not able to run the master node with rkt yet.
```
Comment (jonboulle) [inline docs/getting-started-guides/rkt/README.md]: s/the master component/master components/

**Comment by jonboulle** [comment]
lgtm, thanks @yifan-gu

**Pull request merged**